backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py

```
@@ -1228,6 +1236,8 @@ def _fetch_single_pass_response(
             document_id=document_id,
             run_id=run_id,
             enable_highlight=tool.enable_highlight,
+            enable_word_confidence=tool.enable_word_confidence,
+            doc_id=doc_id,
```

Ruff (0.14.5)

Undefined name `doc_id`

(F821)